Source/WebCore/rendering/svg/RenderSVGInlineText.cpp

```diff
@@ -209,16 +209,19 @@ void RenderSVGInlineText::computeNewScaledFontForStyle(const RenderObject& rende
 {
     // Alter font-size to the right on-screen value to avoid scaling the glyphs themselves, except when GeometricPrecision is specified
     scalingFactor = SVGRenderingContext::calculateScreenFontSizeScalingFactor(renderer);
-    if (!scalingFactor || style.fontDescription().textRenderingMode() == TextRenderingMode::GeometricPrecision) {
+    if (scalingFactor == 1 || !scalingFactor) {
         scalingFactor = 1;
         scaledFont = style.fontCascade();
         return;
     }
 
+    if (style.fontDescription().textRenderingMode() == TextRenderingMode::GeometricPrecision)
+        scalingFactor = 1;
```

Comment:
I do not understand why we have to have a special case for GeometricPrecision here? According to https://developer.mozilla.org/en-US/docs/Web/CSS/text-rendering this property will honor the scaling precision over the speed.